### ISSUE TITLE
How can I extend the original mxGraph prototype?

### ISSUE THREAD

**Issue by awesomenkj** [asual/mxgraph-factory#4]
How can I extend the original mxGraph prototype? 
Something like this. 
```javascript
mxGraph.prototype.pageBreakColor = '#c0c0c0';
....
 var graphUpdatePageBreaks = mxGraph.prototype.updatePageBreaks;
	
mxGraph.prototype.updatePageBreaks = function(visible, width, height)
{
	var useCssTranforms = this.useCssTransforms, scale = this.view.scale, 
		translate = this.view.translate;

	if (useCssTranforms)
	{
		this.view.scale = 1;
		this.view.translate = new mxPoint(0, 0);
		this.useCssTransforms = false;
	}
	
	graphUpdatePageBreaks.apply(this, arguments);
	
	if (useCssTranforms)
	{
		this.view.scale = scale;
		this.view.translate = translate;
		this.useCssTransforms = true;
	}
};
```

**Comment by tbouffard** [comment]
@rabpeter have you been able to fix you issue?
It seems that you have copied/adapted this repository to manage your own solution
  - https://github.com/rabpeter/ts-mxgraph-factory
  - https://github.com/rabpeter/ts-mxgraph

**Comment by awesomenkj** [comment]
@tbouffard Yep. I solved the problem with my own packages and published them on NPM. 
https://www.npmjs.com/package/ts-mxgraph

**Comment by tbouffard** [comment]
@rabpeter cool, good news then.

I am aware of your package because I work on a project that have forked it (https://github.com/process-analytics/ts-mxgraph) to add missing mxGraph type defintions.
That's why I contacted you in https://github.com/rabpeter/ts-mxgraph-typings/issues/1 as I did with the `asual/mxgraph-factory` repository owner (see #9) to know if you would be interested in joining a [dedicated community initiative](https://github.com/process-analytics/mxgraph-road-to-DefinitelyTyped) to provide mxGraph type definitions through the [DefinitelyTyped project](https://definitelytyped.org/).

**Comment by tbouffard** [comment]
@rabpeter hello are you still interested in or working on mxgraph typescript integration?
I would like to talk to you about https://github.com/rabpeter/ts-mxgraph-typings/issues/1
